### ISSUE TITLE
Bug report: Loading never completes 

### ISSUE THREAD

**Issue by Daniel-UCL** [Daniel-GDI-Hub/data-portal#22]
## The problem

Using Chrome on MacOS with Voice Over, the browser keeps saying that "website web content busy".This happens many minutes after the page loaded, as if the page was constantly trying to load more and more info.Very annoying with Voice Over

## Help us reproduce the problem

**Steps to reproduce the behaviour:**

**Add screenshots**

**Desktop (please complete the following information):**

 - OS: [Mac OS
 - Browser: Chrome

**Smartphone (please complete the following information):**

**Additional context**

## The expectation

The page should load and the screen reader should not be interrupted by the status of a loading bar.

**Comment by matthew-gibbs** [comment]
Is this on a specific page, or is this on all pages?

**Comment by Daniel-UCL** [comment]
On all pages I tried. Voice Over on Google Chrome, announces:

"website web content busy heading level 2 Country Summary”,

For most navigation actions, such as navigating by heading. Web content busy, typically means there is a loading process that VO picks up.

**Comment by Daniel-UCL** [comment]
Update to my previous comment. Appears to happen on specific parts of pages, for example the 3 headings between “World study map” and “Quick links” on the home page. Also, this behaviour only happens on the “country summary” heading for country pages, but not the study title headings. Odd. Is there a difference in how these sections are loaded? I can upload a screen share with Voice Over sound shared to illustrate the issue. LMK, if that would help.

**Comment by Saferrei** [comment]
When I tested the voiceover on chrome, this issue did not happen. It might be fixed.

**Comment by Daniel-UCL** [comment]
Sadly, the issue is still present, but it might be a global issue with Chrome and Voice Over, and therefore beyond our control. Some links I found are:

A history of the same Voice Over issue on Chromium, from 2018, 2016, 2012… So I have a feeling a recent Chrome update might have reintroduced the “website web content busy” error with Voice Over. Here is an issue tracker describing the issue.
https://bugs.chromium.org/p/chromium/issues/detail?id=861567

I found a few helpful tips on this page, with regards to loading notifications for screen readers:
https://ux.stackexchange.com/questions/131889/accessible-way-of-notifying-a-screen-reader-about-loading-the-dynamic-web-page-u

Again, it’s a few years old, but I wonder if the “aria-busy=false” might help. There is also mention of focus management, and using a “.focus” and tabindex=“-1” to place the screen reader focus in the right place (e.g. the H1 on a page newly loaded). This might be a small win for the filter modal, showing that there were no matches found for the current filter setting. Something like this, or the many other things they talk about on the page may also help when the AI bot (Johny) is loading.
